docs/errors.rst

```diff
@@ -58,7 +58,7 @@ instance after a processing of a document:
 
   - ``document_error_tree``: A ``dict``-like object that allows you to query
     nodes corresponding to your document.
-    The subscript notation on a node allows to fetch either a specific error
+    The subscript notation on a node allows you to fetch either a specific error
```

Comment:
since there's no second singular around here and the docs usually (?) use `one` as in your other changes, it'd be good to apply that here too for stylistic consistency.